pkg/ui/admin/embed/layouts/main.html

```diff
@@ -6,9 +6,9 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>FastTrackML</title>
   <link rel="stylesheet" href="/static/aim/assets/icomoon/icomoonIcons.css">
-  <link rel="stylesheet" href="/static/chooser/css/simple.min.css">
+  <link rel="stylesheet" href="/static/chooser/static/css/simple.min.css">
   <link rel="stylesheet" href="/admin/static/css/namespaces.css">
-  <link rel="icon" type="image/x-icon" href="/static/chooser/favicon.ico">
+  <link rel="icon" type="image/x-icon" href="/static/chooser/static/favicon.ico">
```

Comment:
`/static/chooser/static` seems like one-too-many `static` directories.

Reply:
Well at least the sources do not have `static/../static` path, but it seems like there'd be a way to avoid this path.